src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -117,13 +120,15 @@ private DownloadMapsWindow(
       progressPanel.download(pendingDownloads);
     }
 
+    pendingDownloads.addAll(ClientContext.DOWNLOAD_COORDINATOR.getDownloads().stream()
```

Comment:
No, terminated means cancelled or successfully finished.
We want to show terminated downloads once to tell the user "hey this download was terminated in the background" (spoiler it's actually not displaying a 100% bar, but you get the point)